### PR TITLE
Addressed compiler errors

### PR DESCRIPTION
**Pull request by spencerjsmall**
…except for .png and .jpg module declarations.

### CODE REVIEW

File: src/contents/all/components/app/App.tsx [hunk 1]
```diff
@@ -223,7 +224,7 @@ const App = () => {
                   loading={loading}
                   log={log}
                   knots={log ? log.knots : []}
-                  createKnot={createKnot}
+                  createKnot={createKnot} // need to include params?
```
Comment (spencerjsmall) [inline src/contents/all/components/app/App.tsx:227]: will take out comment here, sorry!

File: src/contents/all/components/knot/Knots/Knots.tsx [hunk 2]
```diff
@@ -38,7 +41,7 @@ const Knots = (props: Props) => {
       {hasKnots &&
         props.knots.map((knot, index) =>
           <Knot
-            key={knot.id}
+            key={knot.id} //knot has no id?
```
Comment (spencerjsmall) [inline src/contents/all/components/knot/Knots/Knots.tsx:44]: will remove comment

File: server/tsconfig.json [hunk 3]
```diff
@@ -5,11 +5,11 @@
...
 
         /* Strict Type-Checking Options */
-        "strict": true,
+        "strict": false,
 
         /* Additional Checks */
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
+        "noUnusedLocals": false,
+        "noUnusedParameters": false,
```
Comment (spencerjsmall) [inline server/tsconfig.json:12]: did this in both src/tsconfig and server/tsconfig

Comment (joelrojo) [reply server/tsconfig.json:12]: dope sounds good!

File: src/contents/all/components/knot/Knots/Knots.tsx (hunk 2 above)
Comment (joelrojo) [reply src/contents/all/components/knot/Knots/Knots.tsx:44]: dope! once you remove I'll approve and merge 🙌🏽

**Pull request merged**